<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CArreraUI</class>
 <widget class="QMainWindow" name="CArreraUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Arrera</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ArreraTheme.qrc">
    <normaloff>:/IconArrera/ArreraI2024.png</normaloff>:/IconArrera/ArreraI2024.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>601</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="APP">
     <item>
      <widget class="QPushButton" name="IDC_TABLEUR">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/iconTableur.png</normaloff>:/iconBTN/iconTableur.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_TRAITEMENT">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/iconTraitementTexte.png</normaloff>:/iconBTN/iconTraitementTexte.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_APP">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/app-drawer.png</normaloff>:/iconBTN/app-drawer.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_NAVIGATEUR">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/iconNavigateur.png</normaloff>:/iconBTN/iconNavigateur.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_PRESENTATION">
       <property name="maximumSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/iconBTN/iconPresentation.png</normaloff>:/iconBTN/iconPresentation.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>70</width>
         <height>70</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>630</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="IDC_QUIT">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Quitter</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_RELOAD">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Recharger</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_APROPOS">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>A propos</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_PARAMETRE">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>35</height>
        </size>
       </property>
       <property name="text">
        <string>Parametre</string>
       </property>
       <property name="icon">
        <iconset resource="ArreraTheme.qrc">
         <normaloff>:/IconArrera/ArreraInterfaceParametre.png</normaloff>:/IconArrera/ArreraInterfaceParametre.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QFrame" name="FMode">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>150</y>
      <width>601</width>
      <height>331</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>601</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="mode">
      <property name="leftMargin">
       <number>50</number>
      </property>
      <item>
       <widget class="QPushButton" name="IDC_MODE1">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>mode 1</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IDC_MODE2">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>mode 2</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IDC_MODE3">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>mode 3</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IDC_MODE4">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>mode 4</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IDC_MODE5">
        <property name="maximumSize">
         <size>
          <width>500</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>mode 5</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="FAssistant">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>571</width>
      <height>131</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>571</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="IDC_SIX">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ArreraTheme.qrc">
          <normaloff>:/iconBTN/six.png</normaloff>:/iconBTN/six.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IDC_COPILOTE">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ArreraTheme.qrc">
          <normaloff>:/iconBTN/ArreraCopiloteIcon.png</normaloff>:/iconBTN/ArreraCopiloteIcon.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="IDC_RYLEY">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="ArreraTheme.qrc">
          <normaloff>:/iconBTN/Ryley.png</normaloff>:/iconBTN/Ryley.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="IDC_LABELUSER">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>601</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>601</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="IDC_TASKBAR">
       <property name="text">
        <string>Arrera Bar</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>50</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="ArreraTheme.qrc"/>
 </resources>
 <connections/>
</ui>
